packages/legend-query-builder/src/stores/QueryBuilderValueSpecificationBuilder.ts

```diff
@@ -101,20 +119,30 @@ export const buildLambdaFunction = (
     ),
   );
 
-  // build getAll()
-  const getAllFunction = buildGetAllFunction(_class, Multiplicity.ONE);
-
   // build milestoning parameter(s) for getAll()
   const milestoningStereotype = getMilestoneTemporalStereotype(
     _class,
     queryBuilderState.graphManagerState.graph,
   );
-  if (milestoningStereotype) {
-    queryBuilderState.milestoningState
-      .getMilestoningImplementation(milestoningStereotype)
-      .buildGetAllParameters(getAllFunction);
+
```

Comment:
maybe we could document the reason why getAll() doesn't work and getAllVersion() is executable ( just in case query with getAll() will also be executable in the future )